tango/__main__.py

```diff
@@ -750,15 +750,16 @@ def log_and_execute_run():
             cli_logger.info("[green]Starting new run [bold]%s[/][/]", run.name)
 
         executor_output: Optional[ExecutorOutput] = None
-        if step_name is not None:
+        if step_names:
```

Comment:
That check isn't necessary. The for loop on line 755 won't run anyways if `step_names` is empty.

Reply:
yea but the `else` branch here is important.